### DISCUSSION TITLE
Disable mobile web view

### DISCUSSION THREAD

**Discussion by christianTF** [community/community#23422] [New to GitHub]
<p>Hi,</p>
<p>could somebody explain how to disable the web view on <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>?<br>
Since some weeks the web view was “optimized”, therefore I cannot get the commit list anymore.<br>
I tried to use Chrome/iOS’s “View Desktop version” but this is not recognized by <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>.</p>
<p>I need to view the commits as I monitor the team’s check-ins, and for support purposes, I sometimes need to have a quick look to our commits.</p>
<p>Thanks<br>
Christian</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/christiantf">@christianTF</a>! Welcome to the community!</p>
<p>Are you looking for the <strong>Mobile Settings</strong> here in the <a href="https://github.com/settings/admin">Admin Settings</a>? Or have you already checked the <strong>Opt out of mobile pages</strong> checkbox in that section?</p>

**Comment by christianTF** [comment, marked as answer]
<p>Hi,</p>
<p>Thanks for reply.<br>
I already had checked the “Opt out” in my settings (that btw too are not reachable in mobile view), but the opt-out does not change the view.</p>
<p>But, at comparing PC and mobile view, I could figure out that the clock symbol in the mobile web view leads to the commit list. In my mobile web view is no “x commits” string, but only that clock symbol.<br>
It is “hide&amp;seek navigation” on the mobile, but at least the commit list is still available.</p>
<p>Thanks,<br>
Christian</p>

**Comment by JavierSegoviaCordoba** [comment]
<p>I am unable to get the desktop version in my iPhone with both, Chrome and iOS.</p>
<p>It is a problem because the mobile version doesn’t work very well with GitHub Actions and the workflow_dispatch dialog (it overflows the screen…)</p>
